### PR TITLE
Updats popup docs with min/max allowed size

### PR DESCRIPTION
**Pull request by dotproto**
Fixes https://bugs.chromium.org/p/chromium/issues/detail?id=270140

### CODE REVIEW

File: site/en/docs/extensions/reference/action/index.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ In order to use the `chrome.action` API, you need to specify a
   "name": "Action Extension",
   "manifest_version": 3,
   "action": {
-    "default_icon": {
-      "16": "images/icon16.png",
-      "24": "images/icon24.png",
-      "32": "images/icon32.png"
+    "default_icon": {                    // optional
```
Comment (samthor) [inline site/en/docs/extensions/reference/action/index.md:25]: Can you align these comments?

**Comment by samthor** [comment]
You can actually enforce popup size via min-max CSS properties, right?

File: site/en/docs/extensions/reference/action/index.md (hunk 1 above)
Comment (dotproto) [reply site/en/docs/extensions/reference/action/index.md]: Done

**Comment by dotproto** [comment]
> You can actually enforce popup size via min-max CSS properties, right?

Yes. To my knowledge you can use HTML attributes, CSS height/width, and CSS min/max.

File: site/en/docs/extensions/reference/browserAction/index.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ Register your browser action in the [extension manifest][2] like this:
   "name": "My extension",
   ...
   "browser_action": {
```
Comment (awfuchs) [inline site/en/docs/extensions/reference/browserAction/index.md:21]: This still says browser_action

File: site/en/docs/extensions/reference/browserAction/index.md [hunk 3]
```diff
@@ -98,7 +98,8 @@ Set the text and color of the badge using [`browserAction.setBadgeText`][13] and
 ### Popup
 
 If a browser action has a popup, the popup appears when the user clicks the icon. The popup can
```
Comment (awfuchs) [inline site/en/docs/extensions/reference/browserAction/index.md:100]: Do we still say "browser action" in descriptive text?

File: site/en/docs/extensions/reference/action/index.md [hunk 4]
```diff
@@ -123,7 +123,8 @@ chrome.action.setBadgeBackgroundColor(
 
 An action's popup will be shown when the user clicks on the extension's button
```
Comment (awfuchs) [inline site/en/docs/extensions/reference/action/index.md:124]: "extension's button" ... isn't this the action? We should perhaps call this UI element "action button" everywhere; WDYT?

**Review by awfuchs — APPROVED** [review]

File: site/en/docs/extensions/reference/action/index.md (hunk 4 above)
Comment (dotproto) [reply site/en/docs/extensions/reference/action/index.md:124]: Edited to say the "extension's action button".

**Pull request merged**